lib/tapioca/cli.rb

```diff
@@ -321,6 +335,10 @@ def __print_version
       def self.exit_on_failure?
         true
       end
+
+      def set_environment
```

Comment:
Should we move this to [cli_helper](https://github.com/Shopify/tapioca/blob/main/lib/tapioca/helpers/cli_helper.rb) so we can attach a signature to it?

Or maybe this should be called from inside the command and thus defined inside the base [command](https://github.com/Shopify/tapioca/blob/main/lib/tapioca/commands/command.rb)?